### PR TITLE
KFSTP-1804 Making C&G labels more consistent

### PR DESCRIPTION
**Pull request by kkronenb**

### CODE REVIEW

File: work/db/upgrades/5.0.3_cgb/rice_server/kim_upgrade.xml [hunk 1]
```diff
@@ -212,8 +212,8 @@
 			<column name="VER_NBR" value="1" />
 			<column name="PERM_TMPL_ID" value="29" />
 			<column name="NMSPC_CD" value="KFS-AR" />
-			<column name="NM" value="Use Screen CG Transmit Contracts and Grants Invoices Process" />
-			<column name="DESC_TXT" value="Allows users access to screens in the KFS-CG module that are not documents, lookups, inquiries, or batch uploads, i.e. this primarily provides access to CG Transmit Contracts and Grants Invoices Process" />
+			<column name="NM" value="Use Screen CG Transmit Contracts & Grants Invoices Process" />
```
Comment (smith750) [inline work/db/upgrades/5.0.3_cgb/rice_server/kim_upgrade.xml]: this should be an &amp;

File: work/db/upgrades/5.0.3_cgb/rice_server/kim_upgrade.xml [hunk 2]
```diff
@@ -212,8 +212,8 @@
...
+			<column name="DESC_TXT" value="Allows users access to screens in the KFS-CG module that are not documents, lookups, inquiries, or batch uploads, i.e. this primarily provides access to CG Transmit Contracts & Grants Invoices Process" />
```
Comment (smith750) [inline work/db/upgrades/5.0.3_cgb/rice_server/kim_upgrade.xml]: And here.  Do you want me to mark all of them?

File: work/src/org/kuali/kfs/module/ar/report/service/impl/ContractsGrantsInvoiceReportServiceImpl.java [hunk 3]
```diff
@@ -63,39 +63,9 @@
 import org.kuali.kfs.sys.service.NonTransactional;
 import org.kuali.kfs.sys.service.OptionsService;
 import org.kuali.kfs.sys.service.ReportGenerationService;
-import org.kuali.rice.core.api.config.property.ConfigurationService;
```
Comment (smith750) [inline work/src/org/kuali/kfs/module/ar/report/service/impl/ContractsGrantsInvoiceReportServiceImpl.java]: Huh.  We hadn't organized imports before this?

Comment (kkronenb) [inline work/src/org/kuali/kfs/module/ar/report/service/impl/ContractsGrantsInvoiceReportServiceImpl.java]: No, I think eclipse freaked out on me and killed some imports. I need to get them back in.

**Comment by kkronenb** [comment]
ok, this is looking better now.

File: work/db/upgrades/5.0.3_cgb/rice_server/kim_upgrade.xml [hunk 4]
```diff
@@ -38,7 +38,7 @@
 			<column name="ROLE_NM" value="CGB Collector" />
 			<column name="NMSPC_CD" value="KFS-AR" />
 			<column name="DESC_TXT"
-				value="Users performing collection activities, identified as part of an AR billing or processing organization. Collectors have access to Contracts Grants Aging Report, Collection Activity Report, Tickler Report, Dunning Letter Distribution Report for their billing or processing organization, further restricted by customer name starting letter range." />
+				value="Users performing collection activities, identified as part of an AR billing or processing organization. Collectors have access to Contracts & Grants Aging Report, Collection Activity Report, Tickler Report, Dunning Letter Distribution Report for their billing or processing organization, further restricted by customer name starting letter range." />
```
Comment (smith750) [inline work/db/upgrades/5.0.3_cgb/rice_server/kim_upgrade.xml]: still no amp;

**Comment by kkronenb** [comment]
PR updated.

File: work/db/upgrades/5.0.3_cgb/workflow/workflow_document_upgrades.xml [hunk 5]
```diff
@@ -960,7 +960,7 @@
 				AR
 			</parent>
 			<label>
-				Contracts and Grants Invoice
+				Contracts & Grants Invoice
```
Comment (smith750) [inline work/db/upgrades/5.0.3_cgb/workflow/workflow_document_upgrades.xml]: here

File: work/db/upgrades/5.0.3_cgb/workflow/workflow_document_upgrades.xml [hunk 6]
```diff
@@ -1020,7 +1020,7 @@
 				AR
 			</parent>
 			<label>
-				Contracts and Grants Letter of Credit Review
+				Contracts & Grants Letter of Credit Review
```
Comment (smith750) [inline work/db/upgrades/5.0.3_cgb/workflow/workflow_document_upgrades.xml]: here

File: work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsAndGrantsAgingReport.xml [hunk 7]
```diff
@@ -30,7 +30,7 @@
 		<property name="lookupDefinition">
 			<ref bean="ContractsAndGrantsAgingReport-lookupDefinition" />
 		</property>
-		<property name="objectLabel" value="Contracts And Grants Aging Report" />
+		<property name="objectLabel" value="Contracts & Grants Aging Report" />
```
Comment (smith750) [inline work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsAndGrantsAgingReport.xml]: here

File: work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsAndGrantsAgingReport.xml [hunk 8]
```diff
@@ -519,7 +519,7 @@
 		parent="ContractsAndGrantsAgingReport-lookupDefinition-parentBean" />
 
   	<bean id="ContractsAndGrantsAgingReport-lookupDefinition-parentBean" abstract="true" parent="LookupDefinition">
-	    <property name="title" value="Contracts And Grants Aging Report"/>
+	    <property name="title" value="Contracts & Grants Aging Report"/>
```
Comment (smith750) [inline work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsAndGrantsAgingReport.xml]: here

File: work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsGrantsAgingOpenInvoicesReport.xml [hunk 9]
```diff
@@ -25,7 +25,7 @@
     <property name="lookupDefinition">
       <ref bean="ContractsGrantsAgingOpenInvoicesReport-lookupDefinition"/>
     </property>
-    <property name="objectLabel" value="Contracts Grants Aging Open Invoices Report"/>
+    <property name="objectLabel" value="Contracts & Grants Aging Open Invoices Report"/>
```
Comment (smith750) [inline work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsGrantsAgingOpenInvoicesReport.xml]: here

File: work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsGrantsAgingOpenInvoicesReport.xml [hunk 10]
```diff
@@ -153,7 +153,7 @@
   <bean id="ContractsGrantsAgingOpenInvoicesReport-inquiryDefinition" parent="ContractsGrantsAgingOpenInvoicesReport-inquiryDefinition-parentBean"/>
 
   <bean id="ContractsGrantsAgingOpenInvoicesReport-inquiryDefinition-parentBean" abstract="true" parent="InquiryDefinition">
-    <property name="title" value="Contracts Grants Aging Open Invoices Report Inquiry"/>
+    <property name="title" value="Contracts & Grants Aging Open Invoices Report Inquiry"/>
```
Comment (smith750) [inline work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsGrantsAgingOpenInvoicesReport.xml]: y aqui

File: work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsGrantsAgingOpenInvoicesReport.xml [hunk 11]
```diff
@@ -183,7 +183,7 @@
 <!-- Business Object Lookup Definition -->
   <bean id="ContractsGrantsAgingOpenInvoicesReport-lookupDefinition" parent="ContractsGrantsAgingOpenInvoicesReport-lookupDefinition-parentBean"/>
   <bean id="ContractsGrantsAgingOpenInvoicesReport-lookupDefinition-parentBean" abstract="true" parent="LookupDefinition">
-    <property name="title" value="Contracts Grants Aging Open Invoices Report"/>
+    <property name="title" value="Contracts & Grants Aging Open Invoices Report"/>
```
Comment (smith750) [inline work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsGrantsAgingOpenInvoicesReport.xml]: and right here

File: work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsGrantsInvoiceDetail.xml [hunk 12]
```diff
@@ -29,7 +29,7 @@ http://rice.kuali.org/dd http://rice.kuali.org/dd/dd.xsd">
 
 	<bean id="ContractsGrantsInvoiceDetail-parentBean" abstract="true" parent="BusinessObjectEntry">
 		<property name="businessObjectClass" value="org.kuali.kfs.module.ar.businessobject.ContractsGrantsInvoiceDetail" />
-		<property name="objectLabel" value="Contracts and Grants Invoice Detail" />
+		<property name="objectLabel" value="Contracts & Grants Invoice Detail" />
```
Comment (smith750) [inline work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsGrantsInvoiceDetail.xml]: this is another one

File: work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsGrantsInvoiceDocumentErrorLog.xml [hunk 13]
```diff
@@ -230,7 +230,7 @@
 
   	<bean id="ContractsGrantsInvoiceDocumentErrorLog-lookupDefinition" parent="ContractsGrantsInvoiceDocumentErrorLog-lookupDefinition-parentBean"/>
   	<bean id="ContractsGrantsInvoiceDocumentErrorLog-lookupDefinition-parentBean" abstract="true" parent="LookupDefinition">
-    	<property name="title" value="Contracts Grants Invoice Document Error Log Report"/>
+    	<property name="title" value="Contracts & Grants Invoice Document Error Log Report"/>
```
Comment (smith750) [inline work/src/org/kuali/kfs/module/ar/businessobject/datadictionary/ContractsGrantsInvoiceDocumentErrorLog.xml]: here

**Comment by kkronenb** [comment]
PR updated again

File: work/workflow/010_accounts_receivable/AccountsReceivableTransactionalDocuments.xml [hunk 14]
```diff
@@ -333,7 +333,7 @@
 				AR
       		</parent>
 			<label>
-				Contracts and Grants Invoice
+				Contracts & Grants Invoice
```
Comment (smith750) [inline work/workflow/010_accounts_receivable/AccountsReceivableTransactionalDocuments.xml]: here

File: work/workflow/010_accounts_receivable/AccountsReceivableTransactionalDocuments.xml [hunk 15]
```diff
@@ -395,7 +395,7 @@
 				AR
       		</parent>
 			<label>
-				Contracts and Grants Letter of Credit Review 
+				Contracts & Grants Letter of Credit Review
```
Comment (smith750) [inline work/workflow/010_accounts_receivable/AccountsReceivableTransactionalDocuments.xml]: here

File: work/src/org/kuali/kfs/module/ar/document/validation/configuration/AccountsReceivableValidators.xml [hunk 16]
```diff
@@ -303,7 +303,7 @@
 		class="org.kuali.kfs.module.ar.document.validation.impl.PaymentApplicationIsFullBalanceAppliedValidation"
 		abstract="true" />
 
-	<!-- Contracts Grants Invoice Document. adding it here as CGIn will be part 
+	<!-- Contracts & Grants Invoice Document. adding it here as CGIn will be part
```
Comment (smith750) [inline work/src/org/kuali/kfs/module/ar/document/validation/configuration/AccountsReceivableValidators.xml]: Is this one okay?

Comment (kkronenb) [inline work/src/org/kuali/kfs/module/ar/document/validation/configuration/AccountsReceivableValidators.xml]: this one seems fine since it is in a comment.

**Pull request merged**